apps/server/src/modules/room/api/room.controller.ts

```diff
@@ -153,6 +154,21 @@ export class RoomController {
 		return response;
 	}
 
+	@ApiOperation({ summary: 'Move a single board item.' })
+	@ApiResponse({ status: 204 })
+	@ApiResponse({ status: 400, type: ApiValidationError })
+	@ApiResponse({ status: 403, type: ForbiddenException })
+	@ApiResponse({ status: 404, type: NotFoundException })
+	@HttpCode(204)
+	@Patch(':roomId/boards')
```

Comment:
similar, maybe endpoint should abstract type of content
:roomId/content or contentPosition

Reply:
Yes, but I would rather change that when we have other types of content.